ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-rollback-stable.yaml

```diff
@@ -66,6 +66,12 @@ releases:
       architecture: amd64
       channel: candidate
       version: 4.15.0-rc.0
+  oldest-supported-z-rollback:
```

Comment:
I do not think we need this release image setting, just like the existing OCP rollback job, we are using `latest` release image as the rollback target, that means, install with `release:latest` -> upgrade to `release:target` -> rollback to `release:latest`.

Reply:
done